src/hatch_openzim/files_install.py

```diff
@@ -0,0 +1,199 @@
+import os
+import shutil
+import tempfile
+import zipfile
+from pathlib import Path
+from typing import Any, Dict, List
+from urllib.request import urlopen
+
+import toml  # to be replaced by tomllib once only 3.11 and above is supported
+
+from hatch_openzim.shared import log
+
+DEFAULT_OPENZIM_TOML_LOCATION = "openzim.toml"
+
+
+def process(openzim_toml_location: str = DEFAULT_OPENZIM_TOML_LOCATION):
+    """performs openZIM operations on files, i.e. download and extract ZIPs
+
+    Configuration is read from openzim.toml which must be in current directory
```

Comment:
This feels unclear. Can we say that it should be at the root of the hatch project (ie. next to `pyproject.toml`)?